scripts/create.ts

```diff
@@ -29,37 +47,37 @@ const chainLocations = getDirectories(join(__dirname, "../src/chains")).map(
   d => `chains/${d}`
 );
 locations = locations.concat(chainLocations);
-const argv = yargs
-  .command(
-    `$0 <name> --location [--folder]`,
-    `Create a new package in the given location with the provided name.`,
-    yargs => {
-      return yargs
-        .usage(
-          chalk`{hex("#e4a663").bold Create a new package in the given {dim <}location{dim >} with the provided {dim <}name{dim >}.}\n\n` +
-            chalk`{bold Usage}\n  {bold $} {dim <}name{dim >} {dim --}location {dim <}location{dim >} {dim [--folder <folder>]}`
-        )
-        .positional("name", {
-          describe: `          The name for the new package.`,
-          type: "string",
-          demandOption: true
-        })
-        .option("location", {
-          alias: "l",
-          describe: `The location for the new package.`,
-          choices: locations,
-          demandOption: true
-        })
-        .option("folder", {
-          alias: "f",
-          describe: chalk`Optional override for the folder name for the package instead of using {dim <}name{dim >}.`
-        });
-    }
-  )
+
+const argv = yargs(getArgv())
+  .command(`${COMMAND_NAME} <name>`, "", yargs => {
```

Comment:
Since we actually provide `.position("name")` I think this should actually be

```suggestion
  .command("$0", "", yargs => {
```

I also don't think we should have `create` there. That would be like running `npm run create create ...`

Reply:
The `argv` array passed into to yargs, as parsed by (`getArgv`) is `["create"] ,"name", "--location" ...]`, so I'd need to update that to remove one more argument from the start of the parsed npm argv.

I put the command name here so it _isn't_ a catch all (`$0`), but I don't think it matters either way, really.

Reply:
Oh, then it's fine as is